File: problems/beginner/1007/1007.rs
use std::io;

fn get_number_from_stdin() -> i32 {
	let mut num = String::new();

	io::stdin()
		.read_line(&mut num)
		.expect("Error while reading from stdin");

	let num: i32 = num
		.trim()
		.parse()
		.expect("Error while parsing string into number");

	num
}

fn main() {
	let a = get_number_from_stdin();
	let b = get_number_from_stdin();
	let c = get_number_from_stdin();
	let d = get_number_from_stdin();

	println!("DIFERENCA = {}", (a * b) - (c * d));
}

File: problems/beginner/1007/1007.dart
import 'dart:convert';
import 'dart:io';

String input() {
  return (stdin.readLineSync(encoding: utf8) as String);
}

int intInput() {
  return int.parse(input().trim());
}

void main() {
  var a = intInput();
  var b = intInput();
  var c = intInput();
  var d = intInput();

  print('DIFERENCA = ${a * b - c * d}');
}
